### DISCUSSION TITLE
PAL PCR-2D-35 Lighting without WiFi Adapter

### DISCUSSION THREAD

**Discussion by nate-marshall** [tagyoureit/nodejs-poolController#470] [Nixie] [answered]
When my pool lights were installed, I didn't realize that the WiFi module was sold separately ($200). After a little digging I found that the remote that came with it is using the 2.4ghz band and can be paired to the controller by pressing a button on the controller and then another button on the remote. Before heading down the road of either modifying the remote with something like an esp/pi zero or attempting to replay commands from the remote (or worst case dropping $200 on the wifi adapter), I thought I would reach out here to see if anyone has tried tying in their lighting to njcPC/REM without the use of wifi or trying to figure out the commands being pushed over rf. I use home assistant and esp home pretty heavily, but would be nice to try to keep njsPC and HA in sync with the lighting.

**Comment by rstrouse** [comment]
Do you have a PCR-1Z?  If you do, they claim to be able to use the Pentair, Jandy, or Hayward on/off sequences emitted from their controllers.  This means you should be able to use Nixie/REM to get the job done if you wire that controller into a REM relay and identify the light with the proper type in the configuration.

I am resisting my opinions on what I think of the pool lighting industry, including these guys.

**Reply by nate-marshall** [reply]
I have the [PAL PCR-2D-35 Receiver/Driver](https://www.poolsupplyunlimited.com/pool/pal-lighting-42-pcr-2d-35-multi-color-remote-control-transformer/180357p1) with a PCT-1 remote. One interesting note in the description is 
`Color touch technology with OEM cloning via Hayward, Jandy & Pentair automation systems.`  I am not 100% sure what that means but it feels like there might be a way to integrate with the receiver. I took a few pictures of the receiver. The picture with the switch position options doesn't show the last position, but S1=ON, S2=ON is labeled PAL. I am currently on the hunt for more documentation. 


![IMG_20220601_200353](https://user-images.githubusercontent.com/46655553/171527243-9125c708-43c3-4459-ad22-a81a2ba15de1.jpg)
![IMG_20220601_200419](https://user-images.githubusercontent.com/46655553/171527244-c7367482-80ea-4844-91e0-cfc20fc0d731.jpg)
![IMG_20220601_200504](https://user-images.githubusercontent.com/46655553/171527245-e182916d-72bb-4bfd-a2b3-db456f1ce30c.jpg)
![IMG_20220601_200259](https://user-images.githubusercontent.com/46655553/171527247-d61e9e8c-71cc-42e0-ac2f-7214e0037d80.jpg)
![IMG_20220601_200244](https://user-images.githubusercontent.com/46655553/171527248-2618a0e9-0159-428e-b2ef-072b09f0eb47.jpg)

**Comment by rstrouse** [comment]
Here you go.  Be careful of the wiring diagram.  It represents a dead short… remarkable.

https://www.poolsupplyunlimited.com/Products/Manuals/180357_2020619115948.pdf

**Comment by rstrouse** [comment, marked as answer]
Looking at that manual, the proper wiring for it appears to be here.  The PCR-2D is where the 12vac transformer would be.
https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-Standalone-Nixie-Pool-Controller#lighting-circuits

**Reply by nate-marshall** [reply]
Thanks @rstrouse! I will get out there this weekend and see what I can get wired up.
